src/chart/line/poly.ts

```diff
@@ -245,7 +246,15 @@ export class ECPolyline extends Path<ECPolylineProps> {
     }
 
     buildPath(ctx: PathProxy, shape: ECPolylineShape) {
-        const points = shape.points;
+        const points = shape.closed
+            ? [
+                shape.points[shape.points.length - 2],
```

Comment:
What if `shape.points.length` is less than 2?

Reply:
We also need to consider the situation when some of the data is null and when `connectNulls` is set to either `true` or `false`. Please add [test cases](https://github.com/apache/echarts/wiki/How-to-make-a-pull-request#test).